src/layouts/Layout.astro

```diff
@@ -79,8 +79,8 @@ const description = Astro.props.description ?? starpodConfig.description;
 
     <!-- Image -->
     <meta content={imageUrl ?? show.image} property="og:image" />
-    <meta content="summary" name="twitter:card" />
-    <meta content={imageUrl ?? show.image} name="twitter:image:src" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:image" content={imageUrl ?? show.image} />
```

Comment:
What did AI say about this? I think I specifically chose the small summary so the image wouldn't be giant, which I think is what the large one does?

Reply:
Ohh, I've always wanted the large ones, and I guess AI does too. I'll switch it back!